sql/core/src/test/resources/sql-tests/inputs/higher-order-functions.sql

```diff
@@ -59,27 +59,4 @@ select zip_with(ys, zs, (a, b) -> a + size(b)) as v from nested;
 select zip_with(array('a', 'b', 'c'), array('d', 'e', 'f'), (x, y) -> concat(x, y)) as v;
 
 -- Zip with array coalesce
-select zip_with(array('a'), array('d', null, 'f'), (x, y) -> coalesce(x, y)) as v;
```

Comment:
This goes to at the end of this file.
- https://github.com/cloud-fan/spark/pull/11/files#diff-6d243536c0f01b90ae730f8e6745b60dR62